src/com/facebook/buck/apple/AppleDescriptions.java

```diff
@@ -508,7 +509,8 @@ static AppleBundle createAppleBundle(
         Optional<FrameworkDependencies> frameworkDependencies =
             resolver.requireMetadata(
                 BuildTarget.builder(dep)
-                    .addFlavors(FRAMEWORK_FLAVOR)
+                    //this seems wrong
```

Comment:
When including frameworks, I do not want to convert all `apple_library` rules into frameworks by default. I might have a huge library like opencv that would be better linked statically so you can still code strip and don't end up with huge frameworks.

We need to figure out an easy way to switch between libs and frameworks that allows fine grained control and doesn't force your whole tree one way or another.